ui/src/Components/CreateProfile.jsx

```diff
@@ -11,30 +11,119 @@ const H3 = styled.h3`
   margin-left: 50px;
 `;
 
-const CreateProfile = () => {
-    return (
-        <React.Fragment>
-            <NavBar />
-            <div>
-                <H3>Complete Profile</H3>
-            </div>
-            <Form>
+class CreateProfile extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            firstName: '',
+            lastName: '',
+            email: '',
+            number: '',
+            profile_id: '',
+            ref1Name: '',
+            ref1Email: '',
+            ref2Name: '',
+            ref2Email: '',
+        };
+
+        this.handleFirstNameChange = this.handleFirstNameChange.bind(this);
```

Comment:
I think that all of this is fine for now, but we should transition to react hooks at some point. I'll bring it up at the next retro.

Reply:
The current /profiles/create page has only "Name", not "First Name", "Last Name"
This is more a comment to fix the form since having fn & ln is correct form! 
We should add a state for attachments, whether it be boolean or an array of S3 links later

Reply:
@christinajiyunlee the s3 part is part of my story, that's gonna drop soon 👀